### DISCUSSION TITLE
Another geist-ui?

### DISCUSSION THREAD

**Discussion by Coke-Code** [fect-org/fect#258] [Q&A] [answered]
What's kind of the realtionship between fect and geist?

**Comment by nonzzz** [comment, marked as answer]
@Coke-Code As you can see. fect is most same as `geist-ui/core`. Because `geist` no longer maintain vue version. So i made `fect`. Only do the `vue3` version, not react.
But way, fect is more friendly for vue user. I think DX is importance.
